sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOIT.java

```diff
@@ -148,6 +150,16 @@ private void gatherAndPublishMetrics(PipelineResult writeResult, PipelineResult
           return NamedTestResult.create(
               uuid, timestamp, "write_time", (writeEnd - writeStart) / 1e3);
         });
+    suppliers.add(
+        reader -> {
+          double totalBytes = reader.getCounterMetric("write_bytes");
+          return NamedTestResult.create(uuid, timestamp, "write_bytes", totalBytes);
```

Comment:
There's no need to collect bytes/items separately for write & read. If it will be different the PAssert will fail.

Reply:
You're right, I corrected it